lib/ok.ex

```diff
@@ -161,22 +161,39 @@ defmodule OK do
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
       ...>   b <- safe_div(a, 2)
+      ...>   OK.success a + b
+      ...> end
+      {:ok, 6.0}
+
+      iex> OK.with do
+      ...>   a <- safe_div(8, 2)
+      ...>   b <- safe_div(a, 2)
       ...>   {:ok, a + b}
       ...> end
       {:ok, 6.0}
 
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
+      ...>   b = 2.0
+      ...>   OK.success a + b
```

Comment:
From my understanding, the 1.4 requirement for parens only exists for local functions with 0 arity. This stems from it being ambiguous between a local function and a variable. For example, take `foo` in `foo |> bar` - is it a variable or a function? If it were `OK.foo`, then we know it's a (remote) function. But without parentheses, it's ambiguous since it's local.